src/cpp/CMakeLists.txt

```diff
@@ -133,13 +133,29 @@ if(MSVC OR APPLE)
     set(ARCH_DIR ${ARCH_DIR}/${CMAKE_BUILD_TYPE})
 endif()
 
+# Put binaries at the top level for NPM package
+if(CPACK_GENERATOR STREQUAL "NPM")
```

Comment:
Does it make sense to use js bindings from build tree directly? If not, you should enforce `NPM` package generator if `ENABLE_JS` is true.